fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -143,6 +157,7 @@ public String getLatestVersion(String url, Class<? extends Resource> resourceTyp
      */
```

Comment:
Add documentation that says you can only call this for definitional resource types and that otherwise it will throw an IllegalArgumentException

Reply:
An `IllegalArgumentException` will **not** be thrown if the resource isn't definitional. The method will simple return null.